modules/importer/src/test.rs

```diff
@@ -1,14 +1,16 @@
 #![cfg(test)]
 
 use super::model::ImportConfiguration;
-use actix_web::http::StatusCode;
-use actix_web::{test, App};
+use actix_web::{
+    http::{header, StatusCode},
+    test, App,
+};
 use serde_json::json;
 use trustify_common::db::Database;
 
 #[actix_web::test]
 async fn test_default() {
-    env_logger::init();
+    let _ = env_logger::try_init();
```

Comment:
Curious why this is better?

Reply:
Because otherwise tests will fail. Having `init` will run that methods twice (for two different tests) and the second one will fail. That's why the CI just failed, because the logger was already initialized by the first test.